### PR TITLE
fix bugs for file and type.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @ansyral @superyyrrzz @hellosnow @qinezh 
#449

### CODE REVIEW

**Comment by ansyral** [comment]
>         private static void AddFileMapping(FileCollection fileCollection, string baseDirectory, DocumentType type, FileMapping mapping)

remove?

---
Refers to: src/docfx/SubCommands/DocumentBuilderWrapper.cs:361 in 4b60716. [](commit_id = 4b6071632eba4b8a71a1a8eb9f5ae333df680d49, deletion_comment = False)

**Comment by ansyral** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

File: test/docfx.E2E.Tests/DocfxSeedSiteTest.cs [hunk 1]
```diff
@@ -89,9 +89,11 @@ public void TestConceptualPage()
         public void TestReferencePage()
         {
             _driver.Navigate().GoToUrl(_urlHomepage);
+            System.Threading.Thread.Sleep(1000);
```
Comment (qinezh) [inline test/docfx.E2E.Tests/DocfxSeedSiteTest.cs]: >System.Threading.Thread.Sleep(1000); [](start = 12, length = 36)

remove?

**Comment by hellosnow** [comment]
:shipit:

**Comment by qinezh** [comment]
:shipit:

**Pull request merged**